pyanaconda/modules/runtime/runtime_commands.py

```diff
@@ -0,0 +1,178 @@
+
+        self.eula_agreed_changed = Signal()
+        self._eula_agreed = False
+
+    def process_kickstart(self, data):
+        """Process the kickstart data."""
+        self.set_logging_host(data.logging.host)
+        self.set_logging_port(data.logging.port)
+        self.set_rescue(data.rescue.rescue)
+        self.set_rescue_nomount(data.rescue.nomount)
+        self.set_rescue_romount(data.rescue.romount)
+        self.set_eula_agreed(data.eula.agreed)
+
+    def setup_kickstart(self, data):
+        """Set up the kickstart data."""
+        data.logging.host = self._logging_host
+        data.logging.port = self._logging_port
+        data.rescue.rescue = self._rescue
+        data.rescue.nomount = self._rescue_nomount
+        data.rescue.romount = self._rescue_romount
+        data.eula.agreed = self._eula_agreed
+
+    @property
+    def logging_host(self):
+        """The host for logging.
+
+        :return: Logging host address.
+        :rtype: str
+        """
+        return self._logging_host
+
+    def set_logging_host(self, host):
+        """Set the host for logging.
+
+        :param host: Host address for logging.
+        :type host: str
+        """
+        self._logging_host = host
+        self.logging_host_changed.emit()
+        log.debug("Logging host set to: %s", host)
+
+    @property
+    def logging_port(self):
+        """The port for logging.
+
+        :return: Logging port number.
+        :rtype: int
+        """
+        return self._logging_port
+
+    def set_logging_port(self, port):
+        """Set the port for logging.
+
+        :param port: Port number for logging.
+        :type port: int
+        """
+        self._logging_port = port
+        self.logging_port_changed.emit()
+        log.debug("Logging port set to: %s", port)
+
+    @property
+    def rescue(self):
+        """Flag indicating whether rescue mode is enabled.
+
+        :return: True if rescue mode is enabled, else False.
+        :rtype: bool
+        """
+        return self._rescue
+
+    def set_rescue(self, rescue):
+        """Enable or disable rescue mode.
+
+        :param rescue: True to enable, False to disable rescue mode.
+        :type rescue: bool
+        """
+        self._rescue = rescue
+        self.rescue_changed.emit()
+        log.debug("Rescue mode set to: %s", rescue)
+
+    @property
+    def rescue_nomount(self):
+        """Flag for disabling mount in rescue mode.
+
+        :return: True if mounting is disabled in rescue mode, else False.
+        :rtype: bool
+        """
+        return self._rescue_nomount
+
+    def set_rescue_nomount(self, nomount):
+        """Enable or disable mount in rescue mode.
+
+        :param nomount: True to disable, False to enable mount in rescue mode.
+        :type nomount: bool
+        """
+        self._rescue_nomount = nomount
+        self.rescue_nomount_changed.emit()
+        log.debug("Rescue mode nomount set to: %s", nomount)
```

Comment:
nomount is boolean, you need %b

Reply:
Unfortunately it is not possible to log a boolean. It is therefore necessary to retype the boolean

Reply:
Ack sorry for wrong hint